src/commands.ts

```diff
@@ -0,0 +1,9 @@
+import { isNullOrEmpty } from './utils';
+import { Options } from './validate';
+
+export function getLoginCommand(options: Options): string {
+    if (isNullOrEmpty(options.ADMIN_USERNAME)) { throw new Error('Username is required'); };
```

Comment:
It would be clearer to use the exact name of the secret we expect (`ADMIN_USERNAME`) instead of `Username` which users will have to translate to the exact value they need to configure

Reply:
Good catch, I agree.